test/unit/specs/DateInput/typedDates.spec.js

```diff
@@ -32,19 +32,17 @@ describe('DateInput', () => {
   it('emits the date when typed', () => {
     const input = wrapper.find('input')
     wrapper.vm.input.value = '2018-04-24'
+    input.trigger('keypress')
```

Comment:
hmm, I'm getting different tests results locally. This passes for me and I added this line because my tests results were showing that line 103 of DateInput.vue was uncovered. Any pointers how ensuring I get the same results as travis-ci?